apps/fluent-tester/src/RNTester/TestComponents/Button/ButtonIconTest.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from '@fluentui-react-native/button';
+import { Stack } from '@fluentui-react-native/stack';
+import * as React from 'react';
+import { Platform, View, Text } from 'react-native';
+import { stackStyle } from '../Common/styles';
+
+export const ButtonIconTest: React.FunctionComponent<{}> = () => {
+  if (Platform.OS !== ('win32' as any)) {
+    return (
+      <View>
+        <Stack style={stackStyle}>
+          <Button icon={require('./icon_24x24.png')} content="Button with Icon" tooltip="button tooltip" />
+        </Stack>
+      </View>
+    );
+  } else {
+    return (
+      <View>
+        <Stack style={stackStyle}>
+          <Text>Button icon not supported on win32.</Text>
```

Comment:
I just noticed you added this  code. Let's just use this thread for conversation because github is annoying

Reply:
<img width="690" alt="Screen Shot 2020-07-26 at 7 15 46 PM" src="https://user-images.githubusercontent.com/6722175/88497186-74259f00-cf74-11ea-93b6-b3b1be5995cb.png">

After commenting out your code. Looks like you were right.

Reply:
Thanks for checking it out.   Was there any error output indicating what failed on win32?

Reply:
RIght. Forgot to add that. 
![Screen Shot 2020-07-27 at 1 26 22 AM](https://user-images.githubusercontent.com/6722175/88520325-49553e00-cfa8-11ea-98a4-e00263228b44.png)